crates/noirc_abi/src/input_parser/toml.rs

```diff
@@ -9,19 +9,14 @@ pub(crate) fn parse<P: AsRef<Path>>(
 ) -> Result<BTreeMap<String, InputValue>, InputParserError> {
     let path_to_toml = path_to_toml.as_ref();
     if !path_to_toml.exists() {
-        return Err(InputParserError::MissingTomlFile(format!("cannot find input file at located {}, run nargo build to generate the missing Prover and/or Verifier toml files", path_to_toml.display())));
+        return Err(InputParserError::MissingTomlFile(path_to_toml.display().to_string()));
```

Comment:
In the future, I'd probably store this as a PathBuf rather than a String since it represents a filepath